src/schema/v2/__tests__/marketingCollections.test.ts

```diff
@@ -172,4 +172,105 @@ describe("MarketingCollections", () => {
       curatedMarketingCollections: payload,
     })
   })
+
+  it("uses the curated sort for marketing collections", async () => {
+    const query = gql`
+      {
+        marketingCollections(sort: CURATED, size: 2, category: "Gallery") {
+          slug
+        }
+      }
+    `
+
+    const payload = [
+      {
+        slug: "percys-z-collection-1",
+      },
+      {
+        slug: "fiby-z-collection-2",
+      },
+    ]
+
+    const context = {
+      authenticatedLoaders: {},
+    } as any
+
+    context.marketingCollectionsLoader = jest
+      .fn()
+      .mockReturnValueOnce(Promise.resolve({ body: payload }))
```

Comment:
**suggestion**: Instead of wrapping in a promise, you can use mockResolvedValue:

```ts
.mockResolvedValue({ body: marketingCollectionsData })
```